inst/rmarkdown/templates/mortality/skeleton/skeleton.Rmd

```diff
@@ -283,8 +283,24 @@ study_data_cleaned <- study_data_cleaned %>%
          recall_end = as.Date("2018-05-01")
   )
 
-# set inappropriate dates to NA
+# create columns for start and end dates
+study_data_cleaned <- study_data_cleaned %>%
+  find_start_date(contains("born_date"),
```

Comment:
can also just name the variables (quoted or otherwise) or do have to use contains?